scripts/render-vm-startup-script.sh

```diff
@@ -23,125 +23,43 @@
-  git clone $MODEL_VIEWER_REPOSITORY
+  git clone $MODEL_VIEWER_REPOSITORY $MODEL_VIEWER_CHECKOUT_DIRECTORY
 fi
 
-cd ./model-viewer
+pushd $MODEL_VIEWER_CHECKOUT_DIRECTORY
 
-# Update the repo from the origin and reset the tracked branch
+# Update <model-viewer> repo from the origin and reset the tracked branch
 git fetch origin
 git add .
 git reset --hard origin/$MODEL_VIEWER_BASE_BRANCH
 
-# If Filament is still around, delete it so that we are sure to build it
-# cleanly from scratch
-if [ -d /tmp/filament ]; then
-  rm -rf /tmp/filament
-fi
-
-# Install NPM dependencies and update the screenshots
-npm install
-# NOTE: This will cause Filament to be compiled
-npm run update-screenshots
-
-FIDELITY_SCREENSHOT_PATH=./test/fidelity
-NUMBER_CHANGED_SCREENSHOTS=`git ls-files -m $FIDELITY_SCREENSHOT_PATH | wc -l`
-
-# If any screenshots changed, we will be creating a branch and crafting a PR
-if [ "$NUMBER_CHANGED_SCREENSHOTS" -gt "0" ]; then
-  UPDATES_BRANCH="screenshot-updates-`TZ='America/Los_Angeles' date +"%m-%d-%y"`"
-  BRANCH_SHA=`git rev-parse --verify $UPDATES_BRANCH`
-
-  # If we already ran today, we already have a branch by this name so delete it
-  if [ ! -z "$BRANCH_SHA" ]; then
-    git branch -d $UPDATES_BRANCH
-  fi
+popd
 
-  git checkout -b $UPDATES_BRANCH
-  # Only add changes in the fidelity testing subtree
-  git add $FIDELITY_SCREENSHOT_PATH
-
-  COMMIT_MESSAGE="🤖 Render fidelity updates as of `TZ='America/Los_Angeles' date`"
-  git config user.name "$GIT_USER_NAME"
-  git config user.email "$GIT_USER_EMAIL"
-  git commit -m "$COMMIT_MESSAGE"
-  git push -f origin $UPDATES_BRANCH
-
-  PULL_REQUEST_BODY="**NOTE**: This PR was automatically generated. It contains updated render results from third-party renderers for use in our fidelity tests. Please remember to take the following steps when reviewing:\n\n - Review the changed images and take note of any patterns (same renderer, same model etc)\n - Review fidelity test results in the CI build and take note of any reported changes\n\nTo read more about \`<model-viewer>\` fidelity testing and learn how to update these screenshots yourself, check out [this explainer](https://github.com/GoogleWebComponents/model-viewer/blob/master/test/fidelity/README.md).\n\nPlease refer to this [startup script](https://github.com/GoogleWebComponents/model-viewer/blob/master/scripts/render-vm-startup-script.sh) for technical details related to how this PR was produced.";
-
-  # Create the PR via Github's API
-  curl --user "$GITHUB_USER:$GITHUB_ACCESS_TOKEN" \
-    -H "Content-Type: application/json" \
-    --data "{ \"title\": \"$COMMIT_MESSAGE\", \"head\": \"$UPDATES_BRANCH\", \"base\": \"$MODEL_VIEWER_BASE_BRANCH\", \"body\": \"$PULL_REQUEST_BODY\", \"maintainer_can_modify\": true }" \
-    https://api.github.com/repos/googlewebcomponents/model-viewer/pulls
-fi
+# Invoke relevant startup sub-scripts
+source $MODEL_VIEWER_CHECKOUT_DIRECTORY/scripts/provision-ubuntu-xenial-vm.sh
```

Comment:
Why `source` instead of just invoking the scripts?

Reply:
To spare having to set exported environment variables multiple times in different places